kieker-extension-cassandra/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+	id 'java'
+	id 'java-library'
+}
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+	// Use jcenter for resolving your dependencies.
+	// You can declare any Maven/Ivy/file repository here.
+	jcenter()
+	maven {
+		url 'https://oss.sonatype.org/content/repositories/snapshots/'
+	}
+}
+
+dependencies {
+	api project(':kieker-monitoring')
+	api project(':kieker-analysis')
+
+	// This dependency is exported to consumers, that is to say found on their compile classpath.
+	//api 'net.kieker-monitoring:kieker:1.13-SNAPSHOT'
+
+	// This dependency is used internally, and not exposed to consumers on their own compile classpath.
+	implementation 'com.datastax.cassandra:cassandra-driver-core:3.4.0'
+
+	// Use JUnit test framework
```

Comment:
I would suggest to remove the two JUnit lines, since this is not used and comments should not be used as alternative way of saving old source code versions

Reply:
Removed.